tests/test_feedexport.py

```diff
@@ -1765,7 +1765,10 @@ def test_lzma_plugin_preset(self):
     def test_lzma_plugin_filters(self):
         import sys
         if "PyPy" in sys.version:
-            raise unittest.SkipTest("lzma filters doesn't work in PyPy")
+            pypy_version = sys.version.split('\n')[1].split()[1]
+            if pypy_version <= '7.3.1':
```

Comment:
This is not a reliable way to test versions:

```python
>>> '10.0.0' <= '7.3.1'
True
```

See `sys.version_info` usage in the code base, something like that should work.